layouts/index.html

```diff
@@ -359,12 +359,4 @@ <h4 class="mb-1">Events</h4>
 <script src="https://cdn.jsdelivr.net/npm/vega-lite@5.0.0"></script>
 <script src="https://cdn.jsdelivr.net/npm/vega-embed@6.8.0"></script>
```

Comment:
Do we need these vega anymore, since they were removed from the postcovid md ?

Reply:
As discussed this morning these are indeed not required anymore, I removed them in b0219c4